block/slvalidator.go

```diff
@@ -84,8 +82,12 @@ func (v *SettlementValidator) ValidateStateUpdate(batch *settlement.ResultRetrie
 		if errors.Is(checkBatchResult.Error, da.ErrBlobNotIncluded) {
 			return types.NewErrStateUpdateBlobNotAvailableFraud(batch.StateIndex, string(batch.MetaData.DA.Client), batch.MetaData.DA.Height, hex.EncodeToString(batch.MetaData.DA.Commitment))
 		}
+
+		// FIXME: how to handle non-happy case? not returning error?
+		continue
```

Comment:
not sure why the continue at the end of the loop

Reply:
it was continuing before
I've added the `continue` for clarity
as it's not obvious why we do it in case of error (see the fixme above)